contract_addresses_template.env

```diff
@@ -0,0 +1,9 @@
+AUTHORITY_ADDRESS={AUTHORITY_ADDRESS}
```

Comment:
What do you think if we move this file into `bin/contract_addresses_template.env` to avoid polluting the root dir?

I know it's not quite a `bin/` thing but it's used solely by a bin.